packages/grafana-schema/src/raw/dashboard/x/dashboard_types.gen.ts

```diff
@@ -517,17 +550,25 @@ export interface Panel {
    */
   thresholds?: Array<unknown>;
   /**
-   * TODO docs
-   * TODO tighter constraint
+   * Overrides the relative time range for individual panels,
+   * which causes them to be different than what is selected in
+   * the dashboard time picker in the top-right corner of the dashboard. You can use this to show metrics from different
+   * time periods or days on the same dashboard.
+   * The value is formated as time operation like: `now-5m` (Last 5 minutes), `now/d` (the day so far),
+   * `now-5d/d`(Last 5 days), `now/w` (This week so far), `now-2y/y` (Last 2 years).
+   * Note: Panel time overrides have no effect when the dashboard’s time range is absolute.
+   * See: https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/#query-options
    */
   timeFrom?: string;
   /**
    * TODO docs
    */
   timeRegions?: Array<unknown>;
   /**
-   * TODO docs
-   * TODO tighter constraint
+   * Overrides the time range for individual panels by shifting its start and end relative to the time picker.
+   * For example, you can shift the time range for the panel to be two hours earlier than the dashboard time picker setting `2h`.
+   * Note: Panel time overrides have no effect when the dashboard’s time range is absolute.
+   * See: https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/#query-options
```

Comment:
@jdbaldry When we want to link to Grafana docs. What's the best way to do it? Thank you!

Reply:
Hey Ivan, is this in the code itself? As far as I'm aware, there is no alternative to specifying the full URL. If it is in documentation, we would use a Hugo relref for build time link checking. More information about linking in documentation is in https://grafana.com/docs/writers-toolkit/writing-guide/references/.